### PR TITLE
Adds two workbooks for workspaces.

### PR DESCRIPTION
**Pull request by avidhanju**
Adding following workbooks for workspace resource type:

- Workspace Usage
image

- Agent Health
image

### CODE REVIEW

File: Workbooks/Workspace/Agent Health/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Agent Health",
+    "author": "Microsoft",
+    "galleries": [
+        { "type": "workbook", "resourceType": "microsoft.operationalinsights/workspaces", "order": 200 }
```
Comment (acearun) [inline Workbooks/Workspace/Agent Health/settings.json]: "order": 100

File: Workbooks/Workspace/Agent Health/Agent Health.workbook [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+            },
+            {
+              "columnMatch": "AgentType",
+              "formatter": 0,
+              "formatOptions": {
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "Healthy",
```
Comment (acearun) [inline Workbooks/Workspace/Agent Health/Agent Health.workbook]: I don't think you have a 'Healthy' column. Can you clear any unused settings?

File: Workbooks/Workspace/Agent Health/Agent Health.workbook [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+            },
+            {
+              "columnMatch": "ParentId",
+              "formatter": 5,
+              "formatOptions": {
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "OSType",
```
Comment (acearun) [inline Workbooks/Workspace/Agent Health/Agent Health.workbook]: All these columns should have friendlier names: OSType -> OS type, ComputerEnvironment -> Environment or Computer Environment. The order should have environment before OSType

File: Workbooks/Workspace/Agent Health/Agent Health.workbook [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+      },
+      "conditionalVisibility": null,
+      "showPin": true,
+      "name": "query - 2"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "Heartbeat\r\n| where TimeGenerated {TimeRange:query}\r\n| summarize LastHeartbeat = max(TimeGenerated) by Computer, OSType, ComputerEnvironment, Category, Version, RemoteIPCountry\r\n| extend State = iff(LastHeartbeat < ago({UnhealthyCriteria}), '❌ Unhealthy', '✔️ Healthy')",
```
Comment (acearun) [inline Workbooks/Workspace/Agent Health/Agent Health.workbook]: Probably take out the emojis and make the icon come from thresholds

File: Workbooks/Workspace/Agent Health/Agent Health.workbook [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+        "resourceType": "microsoft.operationsmanagement/solutions"
+      },
+      "conditionalVisibility": null,
+      "name": "parameters - 1"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "datatable(State:string, Rank:int)[\"❌ Unhealthy\", 0, \"✔️ Healthy\", 1]\r\n| join kind = leftouter (Heartbeat\r\n| where TimeGenerated {TimeRange:query}\r\n| summarize LastHeartbeat = max(TimeGenerated) by Computer\r\n| extend State = iff(LastHeartbeat < ago({UnhealthyCriteria}), '❌ Unhealthy', '✔️ Healthy')\r\n| summarize Count = count() by State) on State\r\n| extend Count = iff(isempty(State1), 0, Count)\r\n| project-away State1\r\n| extend Rank = iff(State == '❌ Unhealthy' and Count == 0, 2, Rank)\r\n| order by Rank asc\r\n| extend AgentText = 'Agents'",
```
Comment (acearun) [inline Workbooks/Workspace/Agent Health/Agent Health.workbook]: Take out the emojis and make the icon come from thresholds

File: Workbooks/Workspace/Agent Health/Agent Health.workbook [hunk 6]
```diff
@@ -0,0 +1,302 @@
...
+      "showPin": true,
+      "name": "query - 2"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "Heartbeat\r\n| where TimeGenerated {TimeRange:query}\r\n| summarize LastHeartbeat = max(TimeGenerated) by Computer, OSType, ComputerEnvironment, Category, Version, RemoteIPCountry\r\n| extend State = iff(LastHeartbeat < ago({UnhealthyCriteria}), '❌ Unhealthy', '✔️ Healthy')",
+        "size": 3,
+        "exportFieldName": "Table Name",
```
Comment (acearun) [inline Workbooks/Workspace/Agent Health/Agent Health.workbook]: Don't think this export is needed.

File: Workbooks/Workspace/Agent Health/Agent Health.workbook (hunk 5 above)
Comment (avidhanju) [reply Workbooks/Workspace/Agent Health/Agent Health.workbook]: I am fixing the formatting for tiles and thresholds. Will add icons to the template after those changes are in.

**Pull request merged**